lib/shell/shellscript.js

```diff
@@ -44,10 +44,11 @@ class ShellScriptWorker {
     async processWithScript(source, rendition) {
         console.log(`executing shell script ${this.script} for rendition ${rendition.id()}`);
 
-        const errorFile = prepareErrorFile(rendition.directory);
+        const errorFile = await prepareErrorFile(rendition.directory);
+        rendition.mimeInfoPath = await prepareMimeInfoFile(rendition.directory, rendition.id());
```

Comment:
Abstraction/encapsulation: The `mimeInfoPath` should be read by the shell script worker itself, and it would call `rendition.setContentType()` accordingly. The Rendition class should not have to know about the logic of using a file here and its format.